### PR TITLE
Changed all 'Hive' references to properly address the Docs and Forum.…

### LINKED ISSUE #723: USER SUPPORT: Update the first FAQ question with clarification for Hive
## Description
Update the first FAQ question on the Hive user support page, to remove any ambiguities in the "Hive" nomenclature. 

Image

## Why
Removing any confusions about the naming of hive. 

## Tasks
- [ ] Change the question title and description as per the latest naming conventions of the Hive.

### LINKED ISSUE #724: UI: Align the Hive Docs with the latest UI suggestions before the workshop
https://anu365.sharepoint.com/:p:/r/sites/ACCESS-NRI/Shared%20Documents/A5.%20Infrastructure/WebOps/WebOps_Meetings/ACCESS-NRI%20Hive%20updates.pptx?d=w60c9b1b425424ce59b764cea51507acf&csf=1&web=1&e=z9WZyS

### LINKED ISSUE #733: Correct references to Hive Docs and Hive Forum
## Description
Make sure that within the website, any reference to the Hive Docs and the Hive Forum are phrased correctly (`ACCESS-Hive Docs` or `Hive Docs`, and `ACCESS-Hive Forum` or `Hive Forum`).

## Why
To reflect what discussed about the naming convention and not to confuse users about naming.

## Close Criteria
- [ ] Make sure Hive Docs and Hive Forum references are phrased correctly within the Hive Docs website

### PR DESCRIPTION
**Pull request by atteggiani**
Fixes #733.
Fixes #724. 

Fixes #723. 
The image has been removed for now.
A new image might be produced in the future, but I would not deem it necessary for the moment (before the workshop).

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -43,4 +34,4 @@ Please delete options that are not relevant.
 
 When your pull request is ready please [request a review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review). 
 
-Unless there is a specific person you want to review your PR please select the **reviewers team**: `ACCESS-Hive/reviewers`. This ensures the load for reviewing pull requests is shared equitably.
+Unless there is a specific person you want your PR to be reviewd by, please select the **WebOps team**: `ACCESS-NRI/WebOps`. This ensures the load for reviewing pull requests is shared equitably.
```
Comment (KAUR1984) [inline .github/pull_request_template.md:37]: Changed the reviewers to webops team.

File: docs/about/contribute/contribute_on_github.md [hunk 2]
```diff
@@ -13,21 +13,21 @@ For a quick reference on how to use Markdown syntax you can refer to the [Markdo
...
 
 To start with, if you have an [existing SSH key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/checking-for-existing-ssh-keys), then simply [add this key to your github account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account), and you are all set to clone the repository. In case you don't have the SSH keys set up on your local computer, it's easy to [create a new SSH key locally](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent), and then [feel free to link this key to your github account](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account).
 
 <terminal-window>
     <terminal-line data="input">git clone git@github.com:ACCESS-NRI/access-hive.org.au.git</terminal-line>
     <terminal-line></terminal-line>
-    <terminal-line>Cloning into 'access-hive.github.io'...</terminal-line>
+    <terminal-line>Cloning into 'access-hive.org.au'...</terminal-line>
```
Comment (KAUR1984) [inline docs/about/contribute/contribute_on_github.md:30]: Fixes of old repo names as well.

File: docs/about/contribute/index.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-Contributions are encouraged from any member of the community regarding any aspect of the ACCESS-Hive user portal.
+Contributions are encouraged from any member of the community regarding any aspect of the ACCESS-Hive Docs user portal.
 
 ## How can I contribute?
 
 
 <div class="card-container">
-    <a href="https://github.com/ACCESS-Hive/access-hive.github.io/issues/new?assignees=&labels=External&projects=&template=simple-issue-template.md&title=" class="horizontal-card" target="_blank">
+    <a href="https://github.com/ACCESS-NRI/access-hive.org.au/issues/new?assignees=&labels=External&projects=&template=simple-issue-template.md&title=" class="horizontal-card" target="_blank">
```
Comment (KAUR1984) [inline docs/about/contribute/index.md:13]: Stale links.

**Review by KAUR1984 — APPROVED** [review]

**Review by KAUR1984 — APPROVED** [review]

File: mkdocs.yml [hunk 4]
```diff
@@ -1,14 +1,11 @@
-site_name: ACCESS-Hive
+site_name: ACCESS-Hive Docs
```
Comment (KAUR1984) [inline mkdocs.yml:1]: Addressed issue #752.

**Comment by KAUR1984** [comment]
Thanks for your work on this PR @atteggiani ! Looks great to me! Just a minor comment on the PR, if possible, it would be great to have just one issue for one PR. I see this PR is a bigger one and it might be a bit hard to review for someone who is not actively developing on it. 

In case one issue per PR is not possible in some cases, it would be really helpful to have a checklist in the PR description for what issues have been addressed, so everyone else is aware of the changes at a glance, and would not create any new issues for things that have already been addressed in a PR. :) 

I was about to create a new issue on changing the `ACCESS-Hive` heading to `ACCESS-Hive docs`, but realised that this has already been addressed in this PR (and there was no existing issue in the board).

**Comment by KAUR1984** [comment]
If there is an issue that would require a huge PR, we might benefit from breaking that issue into smaller ones :)

**Review by flicj191 — APPROVED** [review]
Thanks for all your work!

**Comment by atteggiani** [comment]
> Just a minor comment on the PR, if possible, it would be great to have just one issue for one PR. I see this PR is a bigger one and it might be a bit hard to review for someone who is not actively developing on it.

Thank you Jasmeen for the comment. 
You are right, it would be ideal to have one single issue for every PR. 
However, in this case, the Issues listed are effectively subissues of the main one (#733).

> In case one issue per PR is not possible in some cases, it would be really helpful to have a checklist in the PR description for what issues have been addressed, so everyone else is aware of the changes at a glance, and would not create any new issues for things that have already been addressed in a PR. :)

Yeah I agree. If a situation like this happens again in the future (very likely), I will make the description of the PR more detailed! :)  

> I was about to create a new issue on changing the `ACCESS-Hive` heading to `ACCESS-Hive docs`, but realised that this has already been addressed in this PR (and there was no existing issue in the board).

Again, there is no specific issue because I thought it would be better (and cleaner) to address the naming problems all at once (so as not to forget anything and because it's the same conceptual fix).

**Pull request merged**